product-web-app/app.ts

```diff
@@ -38,13 +37,18 @@ app.set('view engine', 'html');
 app.use('/public', express.static(path.join(dirname, 'public')));
 
 app.listen(3000, () => {
+  // eslint-disable-next-line no-console
   console.log('Server listening on port 3000');
 });
 
-// Routing
-//
-// app.get('/', (eq: Request, res: Response) => {
-//   res.redirect('/products');
-// });
+app.get('/', async (req: Request, res: Response) => {
+  if (!req.session.token || req.session.token.length === 0) {
+    res.redirect('auth/login');
+  } else {
+    res.render('index', { title: 'Main page' });
+  }
+});
```

Comment:
Thats not a part of this story
```suggestion
    res.render('index', { title: 'Main page' });
});
```